hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/EntityContractTypeMigration.java

```diff
@@ -0,0 +1,54 @@
+package com.hedera.mirror.importer.migration;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.google.common.base.Stopwatch;
+import com.google.common.collect.Iterables;
+import java.util.Collection;
+import javax.inject.Named;
+import lombok.CustomLog;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Lazy;
+
+import com.hedera.mirror.importer.repository.EntityRepository;
+
+@CustomLog
+@Named
+@RequiredArgsConstructor(onConstructor_ = {@Lazy})
```

Comment:
Lazy not needed here since there's no potential for circular beans. There might be a reflection penalty for keeping `@Lazy` that we don't want to pay.

```suggestion
@RequiredArgsConstructor
```

Reply:
Got it I have removed `@Lazy`